<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SchedulePrintDialogUI</class>
 <widget class="QDialog" name="SchedulePrintDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Calendar Print Options</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Page Layout</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>4</number>
         </property>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Vertical Pages:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QComboBox" name="mOrientationCombo" >
           <item>
            <property name="text" >
             <string>Landscape</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Portrait</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Page Size:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QSpinBox" name="mVerticalPagesSpin" >
           <property name="minimum" >
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QSpinBox" name="mHorizontalPagesSpin" >
           <property name="maximum" >
            <number>50</number>
           </property>
           <property name="minimum" >
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QComboBox" name="mPageSizeCombo" />
         </item>
         <item row="3" column="0" >
          <widget class="QLabel" name="label_8" >
           <property name="text" >
            <string>Orientation:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Horizontal Pages:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Pdf Options</string>
         </property>
         <property name="checkable" >
          <bool>false</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>6</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="mCustomPageSizeLabel" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>Custom Page Size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mCustomPageSizeXSpin" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6" >
              <property name="text" >
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="mCustomPageSizeYSpin" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mCustomPageSizeScaleCombo" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <item>
               <property name="text" >
                <string>Inches</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>MM</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Dpi:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="mDpiSpin" >
              <property name="maximum" >
               <number>10000</number>
              </property>
              <property name="minimum" >
               <number>10</number>
              </property>
              <property name="value" >
               <number>72</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Display Options</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="mAssetTreeGroup" >
         <property name="title" >
          <string>Asset Tree</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>6</number>
          </property>
          <property name="spacing" >
           <number>2</number>
          </property>
          <item>
           <widget class="QRadioButton" name="mNoTreeRadio" >
            <property name="text" >
             <string>None</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mLeftTreeRadio" >
            <property name="text" >
             <string>Left Pages</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mEveryTreeRadio" >
            <property name="text" >
             <string>Every Page</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mDateHeaderGroup" >
         <property name="title" >
          <string>Date Header</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>6</number>
          </property>
          <property name="spacing" >
           <number>2</number>
          </property>
          <item>
           <widget class="QRadioButton" name="mNoHeaderRadio" >
            <property name="text" >
             <string>None</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mTopHeaderRadio" >
            <property name="text" >
             <string>Top Pages</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="mEveryHeaderRadio" >
            <property name="text" >
             <string>Every Page</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="title" >
          <string>Fonts</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>6</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="mFontTree" />
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5" >
      <attribute name="title" >
       <string>Header/Footer</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="mHeaderGroup" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Header</string>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>6</number>
          </property>
          <property name="spacing" >
           <number>4</number>
          </property>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="mRightHeaderText" />
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="mMiddleHeaderText" />
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="mLeftHeaderText" />
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Right Text:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Middle Text:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Left Text:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mFooterGroup" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>5</hsizetype>
           <vsizetype>5</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title" >
          <string>Footer</string>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Left Text:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="mMiddleFooterText" />
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="mRightFooterText" />
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="mLeftFooterText" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_11" >
            <property name="text" >
             <string>Middle Text:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_12" >
            <property name="text" >
             <string>Right Text:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>414</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="mPreviewButton" >
       <property name="text" >
        <string>Pdf Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mSavePdfButton" >
       <property name="text" >
        <string>Save Pdf...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mPrintButton" >
       <property name="text" >
        <string>Print...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mCloseButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mPrintButton</sender>
   <signal>clicked()</signal>
   <receiver>SchedulePrintDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>SchedulePrintDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
